src/bundle/Resources/views/themes/admin/ui/field_type/edit/ezimage.html.twig

```diff
@@ -45,12 +45,18 @@
                         </svg>
                         <span class="ibexa-btn__label">{{ 'content.field_type.ezimage.preview'|trans|desc('Preview') }}</span>
                     </a>
+
+                    {% if version_info is defined %}
+                        {% set content_id = version_info.contentInfo.id %}
+                        {% set version_no = version_info.versionNo %}
+                    {% endif %}
+
                     {{ ibexa_render_component_group(
                         'image-edit-actions-after',
                         {
                             'fieldDefinitionIdentifier' : form.parent.vars.value.fieldDefinition.identifier,
-                            'contentId' : app.request.get('contentId'),
-                            'versionNo' : app.request.get('versionNo'),
+                            'contentId' : content_id|default(app.request.get('contentId')),
+                            'versionNo' : version_no|default(app.request.get('versionNo')),
```

Comment:
Personally, I prefer null-coalescing operator over `default` filter (that also uses loose comparison, not strict `null` check).
```suggestion
                            'contentId' : content_id ?? app.request.get('contentId'),
                            'versionNo' : version_no ?? app.request.get('versionNo'),
```